### PR TITLE
Update GA satisfaction events

### PR DESCRIPTION
**Pull request by colinbruce**
Add a label containing the satisfaction text to the GA event handler

### CODE REVIEW

File: app/controllers/feedback_controller.rb [hunk 1]
```diff
@@ -12,7 +12,7 @@ def new
   def create
     @feedback = Feedback.new(merged_feedback_params)
     if @feedback.save
-      GoogleAnalytics::Api.delay.event('satisfaction', merged_feedback_params[:rating], params[:ga_client_id])
+      submit_feedback_google_event if @feedback.is?'feedback'
```
Comment (csutter) [inline app/controllers/feedback_controller.rb]: What's Rubocop saying about that? 🤖

**Review by csutter — DISMISSED** [review]
:+1:

File: app/controllers/feedback_controller.rb (hunk 1 above)
Comment (colinbruce) [reply app/controllers/feedback_controller.rb]: image
I checked first! :)

**Review by csutter — APPROVED** [review]
:+1:

**Pull request merged**